http_client.rst

```diff
@@ -984,9 +1000,10 @@ To force HTTP/2 for ``http`` URLs, you need to enable it explicitly via the
 
         $client = HttpClient::create(['http_version' => '2.0']);
 
-Support for HTTP/2 PUSH works out of the box when libcurl >= 7.61 is used with
-PHP >= 7.2.17 / 7.3.4: pushed responses are put into a temporary cache and are
-used when a subsequent request is triggered for the corresponding URLs.
+Support for HTTP/2 PUSH works out of the box when using ``amphp/http-client``
```

Comment:
I suggest using using `Support for HTTP/2 PUSH works out of the box when using a compatible client`, as the compatibility requirements are already documented a few lines above. It would avoid maintaining that info twice.